.travis.yml

```diff
@@ -1,13 +1,13 @@
 dist: trusty
-#if: branch = master
+if: branch = master

```

Comment:
I guess we still want the jobs to be executed just no push to docker hub

Reply:
Yes, I will fix it.

Reply:
I also remove this condition via the last commit